swagger.yaml

```diff
@@ -545,6 +547,60 @@ paths:
           description: Version with given `version_id` not found
           content: {}
       x-codegen-request-body-name: body
+  "/models/{model_id}/versions/{version_id}/image":
+    get:
+      tags:
+      - version_image
+      summary: Get the Docker image for given model version
+      parameters:
+      - name: model_id
+        in: path
+        required: true
+        schema:
+          type: integer
+      - name: version_id
+        in: path
+        required: true
+        schema:
+          type: integer
+      responses:
```

Comment:
Should we also include the a `400` BadRequest (https://github.com/caraml-dev/merlin/blob/b5733e50dec330314f0cf30c02a4fdccf475e903/api/api/response.go#L118) error when the model type is invalid, and a `500` (https://github.com/caraml-dev/merlin/blob/b5733e50dec330314f0cf30c02a4fdccf475e903/api/api/response.go#L127) error when the appropriate image builder cannot be found?